internal-services/catalog/create-advisory-task.yaml

```diff
@@ -41,6 +41,7 @@ spec:
       description: The advisory url if the task succeeds, empty string otherwise
   steps:
     - name: create-advisory
+      # TODO: Update this once https://github.com/redhat-appstudio/release-service-utils/pull/129 is merged
```

Comment:
Adding a note here to update the image before merging this.

Reply:
https://github.com/redhat-appstudio/release-service-utils/pull/129 is merged, so I updated the image.